language-server/modules/langserver-core/src/test/resources/command/source/testgen/module2/services.bal

```diff
@@ -0,0 +1,74 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/websub;
+
+service<http:Service> httpService bind { port: 9090 } {
+    sayHello(endpoint caller, http:Request request) {
+    }
+}
+
+service<http:Service> httpsService bind securedListener {
+    sayHello(endpoint caller, http:Request request) {
+    }
+}
+
+service<http:WebSocketService> wsService bind { port: 9094 } {
+    onOpen(endpoint caller) {
+
+    }
+    onText(endpoint caller, string text, boolean final) {
+        io:println("received: " + text);
+    }
+    onClose(endpoint caller, int statusCode, string reason) {
+
```

Comment:
Shall we remove this extra line from the resource body? Also check for other places which has this extra line inside the resource body